qemu/tests/blk_commit.py

```diff
@@ -50,3 +55,24 @@ def create_snapshots(self):
                 logging.info("expect file: {0}, opening file: {1}".format(snapshot, image_file))
                 self.test.fail("create snapshot '%s' failed" % snapshot)
             self.trash_files.append(snapshot)
+            #create file inside guest after each snapshot
+            self.create_file("%s_%s" % (guest_file_name, snapshot_name))
```

Comment:
Why create file when create snapshots ?

Reply:
Otherwise live commit will commit nothing, it will only commit an empty snapshot, that's not the test cases want.